.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea/
 .vscode/
-
-venv/
\ No newline at end of file
+dist/
+venv/
+text_localization_environment.egg-info/
```

Comment:
Please add a trailing newline to the file. 🙂 (I know that it hasn't been there before 😉)

Reply:
Fixed